src/content/en/showcase/2016/iowa2016.md

```diff
@@ -20,7 +20,7 @@ description: Building the Google I/O 2016 PWA using web components
 
 ## TL;DR
 
-Learn how we built a SPA using web components, Polymer, and material design and launched it into production on Google.com.
+Learn how we built a single page app using web components, Polymer, and Material Design and launched it into production on Google.com.
```

Comment:
100% sure it should be b/c they dont want it to be a proper noun. It's a concept/type of design. https://material.google.com/ has lowercase.